docs/recipes/design-patterns/component-composition.md

```diff
@@ -0,0 +1,133 @@
+# Component composition/extending
+
+Since Skate is already close to the platform, you can easily extend other component classes in 2 ways:
+
+- via [Inheritance](#inheritance)
+- via [Mixins](#mixins)
```

Comment:
It might be nice to add decorators to this list at some point. Doing so would mean we need to update this section's title and the description under it, too.